### PR TITLE
Handling Devices with Firmware >= 5.8 but no humidity and pressure sensor

### PR DESCRIPTION
**Pull request by pmneo**
Added Debug Output
Handling Devices with Firmware >= 5.8 but no humidity and pressure sensor
Fixed Sensor Tab Mapping

### CODE REVIEW

File: indi-aagcloudwatcher-ng/CloudWatcherController_ng.cpp [hunk 1]
```diff
@@ -143,34 +143,38 @@ bool CloudWatcherController::getAllData(CloudWatcherData *cwd)
 
         if (!check)
         {
+            LOG_DEBUG( "ERROR in getIRSkyTemperature" );
```
Comment (knro) [inline indi-aagcloudwatcher-ng/CloudWatcherController_ng.cpp:146]: LOG_ERROR

File: indi-aagcloudwatcher-ng/CloudWatcherController_ng.cpp [hunk 2]
```diff
@@ -143,34 +143,38 @@ bool CloudWatcherController::getAllData(CloudWatcherData *cwd)
...
             return false;
         }
 
         check = getIRSensorTemperature(&sensorTemperature[i]);
 
         if (!check)
         {
+            LOG_DEBUG( "ERROR in getIRSensorTemperature" );
```
Comment (knro) [inline indi-aagcloudwatcher-ng/CloudWatcherController_ng.cpp:154]: Same as above

**Review by knro — CHANGES_REQUESTED** [review]
Use LOG_ERROR or LOGF_ERROR to report errors. Please update as well Debian changelog.

**Comment by pmneo** [comment]
Is this correct?

indi-aagcloudwatcher-ng (1.7) focal; urgency=medium

  * Added Debug Output
  * Handling Devices with Firmware >= 5.8 but no humidity and pressure sensor
  * Fixed Sensor Tab Mapping

 -- Philip Mair  <philip@pmneo.de>  Sun, 7 August 2022 12:00:00 +0200

CS

**Pull request merged**